abrt.spec.in

```diff
@@ -357,45 +346,15 @@ Requires: python3-argh
 Requires: python3-argcomplete
 Requires: python3-humanize
 
-%description cli-ng
-New generation command line interface for ABRT
-%endif # with python3
+Provides:  %{name}-cli-ng = %{version}-%{release}
+Obsoletes: %{name}-cli-ng <= 2.12.1
 
-%package cli
-Summary: Virtual package to make easy default installation on non-graphical environments
-Requires: %{name} = %{version}-%{release}
-Requires: abrt-tui
-Requires: abrt-addon-kerneloops
-Requires: abrt-addon-pstoreoops
-Requires: abrt-addon-vmcore
-Requires: abrt-addon-ccpp
-%if %{with python3}
-Requires: python3-abrt-addon
-%endif # with python3
-Requires: abrt-addon-xorg
-%if 0%{?rhel}
-Requires: libreport-rhel >= %{libreport_ver}
-Requires: libreport-plugin-rhtsupport >= %{libreport_ver}
-%else
-%if %{with retrace}
-Requires: abrt-retrace-client
-%endif
-%if %{with bodhi}
-Requires: abrt-plugin-bodhi
-%endif
-%if 0%{!?suse_version:1}
-Requires: libreport-plugin-bugzilla >= %{libreport_ver}
-%endif
-Requires: libreport-plugin-logger >= %{libreport_ver}
-Requires: libreport-plugin-ureport >= %{libreport_ver}
-%if 0%{?fedora}
-Requires: libreport-fedora >= %{libreport_ver}
-%endif
-%endif
+Provides:  %{name}-tui = %{version}-%{release}
+Obsoletes: %{name}-tui <= 2.12.1
```

Comment:
So this came up yesterday - is the “provides” line correct?

I would say that I should keep the subpackage as is for the purposes of this PR, but the idea behind that was that we shouldn’t offer non-ABRT interfaces in the ABRT package.